barretenberg/cpp/src/barretenberg/translator_vm/translator_circuit_builder.cpp

```diff
@@ -405,7 +405,7 @@ void TranslatorCircuitBuilder::populate_wires_from_ultra_op(const UltraOp& ultra
     auto& op_wire = std::get<WireIds::OP>(wires);
     op_wire.push_back(add_variable(ultra_op.op_code.value()));
     // Similarly to the ColumnPolynomials in the merge protocol, the op_wire is 0 at every second index
-    op_wire.push_back(zero_idx);
+    op_wire.push_back(add_variable(0));
```

Comment:
one downside to this vs using zero_idx is that it increases memory usage. It seems perfectly safe to me to use zero_idx but perhaps we should make it so that one cannot change zero_idx once it has been set once and for all. i.e. make it private and accessible only via a getter.

Reply:
https://github.com/AztecProtocol/barretenberg/issues/1534